index.js

```diff
@@ -6206,5 +6206,6 @@ export async function submitBom(args, bomContents) {
       console.log("Unable to submit the SBOM to the Dependency-Track server");
       console.log(error);
     }
+    return error.response;
```

Comment:
Thank you. Could you kindly update the jsdoc in line 6120 to use `@returns`? And then run `npm run gen-types`

Reply:
Done, but formatting is different.
I used nvm with Node v20.13.1 and reinstalled all my node_modules, but still the formatting is different.
I excluded every other "types" file from the PR to prevent committing only formatting on those files.

Reply:
On a similar topic, would you be OK to use prettier (or any other formatting tool) on the project?

Reply:
OK, I just saw that you're using biome. I applied formatting. But when running "gen-types" then "lint", isn't there a risk that `.d.ts.map` files are not mapping correctly (if `.d.ts` files are reformatted *after* the map is generated)?

Reply:
We use biome. You can run `npm run lint` to auto format.

Reply:
Yes I saw. But if we run:
- `npm run gen-types`: it generates `.d.ts` and `.d.ts.map` files
- `npm run lint` : it formats `.d.ts`. If I'm not mistaking, because `.d.ts` files have be reformatted, the referenced line numbers in `.d.ts.map` file are not matching the line numbers in `.d.ts` file because reformatting changed them.

Reply:
@marob That is a great observation. Do you have any recommendation to fix this? @setchy any thoughts?

Reply:
Not fully following, but we have `ignore": ["types/**"]` in biome.js for the linting options.  We can also add that to the formatter options

Reply:
https://github.com/CycloneDX/cdxgen/pull/1110